node_modules/.bin/acorn.cmd

```diff
@@ -0,0 +1,17 @@
+@ECHO off
+GOTO start
+:find_dp0
+SET dp0=%~dp0
+EXIT /b
+:start
+SETLOCAL
+CALL :find_dp0
+
+IF EXIST "%dp0%\node.exe" (
+  SET "_prog=%dp0%\node.exe"
```

Comment:
- [ ] You should never push this folder `node_modules` to GitHub because it contains very large files. [Read this article to get a better understanding](https://flaviocopes.com/should-commit-node-modules-git/).
To fix this, you will have to manually delete it from GitHub. Then, add a file name `.gitignore`, in the root of your project. After that, add the node_modules folder name to it. This will prevent you from accidentally pushing it to Github in subsequent commits. [How to add `.gitignore`](https://www.codegrepper.com/code-examples/whatever/how+to+add+gitignore+for+node_modules).

Kindly fix this and re-submit your project.